scripts/expect/ping.exp

```diff
@@ -15,10 +15,16 @@ TEST_CASE_TARGET {Embox should ping host} {
 	return 0
 }
 
-TEST_CASE_TARGET {Embox should ping google.com} {
-	variable host_ip
+TEST_CASE_TARGET {Embox should ping peer host by IP} {
+	send "ping -c 4 " $env(PEER_HOST_IP) "\r"
+	test_assert_regexp_equal "0 errors"
+
+	return 0
+}
```

Comment:
Let's start the discussion here.

> I've tried, not helped.
Are both forms valid?

> 	send "ping -c 4 " $env(PEER_HOST_IP) "\r"
	send "ping -c 4 $env(PEER_HOST_IP)\r"

I am not sure about the first one, but the second one is valid.

The variant I proposed is:
```
send "ping -c 4 $::env(PEER_HOST_IP)\r"
```

it works for me locally.

Reply:
I didn't check all the things with scripts/continuous/run.sh, but checked that environment variable `PEER_HOST_IP`, which I exported in a shell script is visible then in ping.exp

```
#!/bin/bash

export PEER_HOST_IP=10.0.2.10

expect scripts/expect/framework/run_all.exp scripts/expect/x86_net_tests.config 10.0.2.16 10.0.2.10 ""
```

Reply:
I thought I tried this way, but apparently I haven't tested it carefully enough. This works, thank you!